docker-compose-k6.account-api.yaml

```diff
@@ -6,7 +6,7 @@ services:
     build:
       context: .
       dockerfile: ./Docker/Dockerfile.account
-    command: ['api']
+    command: ['account-api']
```

Comment:
Other places we removed the command like in `docker-compose-k6.content-publishing-api.yaml`. Not sure which is correct.

Reply:
This is correct for this particular service. Not sure why it didn't follow the same pattern; I'd like to address Docker consolidation in a future PR.